components/inputs/test/input-date.visual-diff.js

```diff
@@ -1,576 +0,0 @@
-
-						});
-
-						it('right arrow', async function() {
-							await openKey(page, '#min-max');
-							await page.keyboard.press('ArrowRight');
-							const rect = await getRect(page, '#min-max');
-							await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-						});
-					});
-
-					describe('value after max same month as max', () => {
-						before(async() => {
-							await page.$eval('#min-max', (elem) => {
-								const input = elem.shadowRoot.querySelector('d2l-input-text');
-								input.value = '02/12/2019';
-							});
-						});
-
-						after(async() => {
-							await page.$eval('#min-max', (elem) => {
-								elem.blur();
-							});
-						});
-
-						it('left arrow', async function() {
-							await openKey(page, '#min-max');
-							await page.keyboard.press('ArrowLeft');
-							const rect = await getRect(page, '#min-max');
-							await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-
-						});
-
-						it('right arrow', async function() {
-							await openKey(page, '#min-max');
-							await page.keyboard.press('ArrowRight');
-							const rect = await getRect(page, '#min-max');
-							await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-						});
-					});
-				});
-
-			});
-		});
-
-		describe('with empty-text', () => {
-
-			afterEach(async() => await reset(page, '#empty-text'));
-
-			it('open', async function() {
-				await open(page, '#empty-text');
-				const rect = await getRect(page, '#empty-text');
-				await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-			});
-
-			it('open with enter', async function() {
-				await openKey(page, '#empty-text');
-				const rect = await getRect(page, '#empty-text');
-				await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-			});
-
-			it('open with click', async function() {
-				await openClick(page, '#empty-text');
-				const rect = await getRect(page, '#empty-text');
-				await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-			});
-		});
-
-		describe('with placeholder', () => {
-
-			afterEach(async() => await reset(page, '#placeholder'));
-
-			it('open', async function() {
-				await open(page, '#placeholder');
-				const rect = await getRect(page, '#placeholder');
-				await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-			});
-
-			it('open with enter', async function() {
-				await openKey(page, '#placeholder');
-				const rect = await getRect(page, '#placeholder');
-				await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-			});
-
-			it('open with click', async function() {
-				await openClick(page, '#placeholder');
-				const rect = await getRect(page, '#placeholder');
-				await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-			});
-		});
```

Comment:
Seems like these ones weren't moved, was that on purpose?

Reply:
Oh yep sorry, I removed them purposefully because it didn't seem worth it to test what the opened dropdown looked like based on it having a placeholder or empty text... they were all the same.